package.json

```diff
@@ -68,5 +68,8 @@
     "ansi-regex@>2.1.1 <5.0.1": ">=5.0.1",
     "glob-parent@<5.1.2": ">=5.1.2",
     "minimatch@<3.0.5": ">=3.0.5"
+  },
```

Comment:
this change should not be necessary, because the front end packages are provided in td.vue/package.json

Reply:
Please Check PR @jgadsden